sds_data_manager/stacks/instrument_lambdas.py

```diff
@@ -60,7 +61,7 @@ def __init__(
             VPC into which to put the resources that require networking.
         """
 
-        super().__init__(scope, construct_id)
+        super().__init__(scope, construct_id, **kwargs)
 
         # Batch Job Inputs
         job_definition_arn = batch_resources.job_definition.attr_job_definition_arn
```

Comment:
Did this line give you error when you deploy? It was giving me error before and I couldn't figure out why it was.